### PR TITLE
Feature/cc 0101 refactor courts codes

### PR DESCRIPTION
**Pull request by stevenliu986**
update tile amount when changing border width.

### CODE REVIEW

File: src/components/PriceBar/TileColorBoard.tsx [hunk 1]
```diff
@@ -7,7 +7,8 @@ import priceFormat from "@/utils/priceFormat";
 
 const TileColorBoard: React.FC = () => {
   const tileBlocks = useStoreSelector((state) => state.priceBar.blocks);
-  const court = useStoreSelector((state) => state.courtSize);
+  // const court = useStoreSelector((state) => state.courtSize);
```
Comment (tatoMa) [inline src/components/PriceBar/TileColorBoard.tsx:10]: It seems replaced by the next line. Please remove it if not using or add a TODO.

**Review by tatoMa — APPROVED** [review]
Awesome work you have done in this PR. Finally, see a custom hook largely used in components. 

The only problem is, this PR is too big, contains 30 files changed. Maybe consider refactoring by steps(such as custom hooks for court components first, other area components second or so)  That will make it easier to read and review. Overall, nice work.

**Comment by tatoMa** [comment]
Now, another question. How can we refactor the redundancy HTML part of all court components? (The UI part)

**Review by EmilyGong111 — COMMENTED** [review]
Screen Shot 2022-07-25 at 1 30 10 pm
It is a little bit weird when I select pro full court, the border width operator is there, but it doesn't allow me to modify the border width.

**Review by Beryl-Zhou — APPROVED** [review]

**Comment by stevenliu986** [comment]
> Screen Shot 2022-07-25 at 1 30 10 pm
> 
> It is a little bit weird when I select pro full court, the border width operator is there, but it doesn't allow me to modify the border width.

Pro Full Court's border is fixed,

**Comment by EmilyGong111** [comment]
> 

How does the user know the border size of pro full court is fixed?

**Pull request merged**